Add notes about edge cluster update limitations (#1904)
spectrocloud/librarium: docs/docs-content/clusters/cluster-management/cluster-updates.md

## Prerequisites

- An active Kubernetes cluster in Palette. 

## Enablement

<Tabs groupId="update-method">
<TabItem value="profile-version" label="Use a new cluster profile version">

This is the recommended best practice for updating a cluster in Palette. 

1. Log in to [Palette](https://console.spectrocloud.com).

2. Navigate to the left **Main Menu** and select **Profiles**.

3. Create a new version of the profile you want to update. For more information, refer to [Version a Cluster Profile](../../profiles/cluster-profiles/modify-cluster-profiles/version-cluster-profile.md).

4. Navigate to the left **Main Menu** and select **Cluster**. 

5. Select the cluster you want to update, and navigate to the **Profile** tab of the cluster. 

6. Next to the name of the profile, click on the version number and select the new version you just published. 

7. Click **Save**. Depending on the changes you made to the profile, doing so might trigger a cluster repave warning. Click **Continue** to confirm updating the cluster. 

</TabItem>

<TabItem value="update-cluster-profile" label="Update cluster profile">

> You can make updates to a profile that is in-use by one or more active clusters, and doing so will trigger an update to all clusters that are using the cluster profile. For more information, refer to [Update a Cluster Profile](../../profiles/cluster-profiles/modify-cluster-profiles/update-cluster-profile.md). 

Reviewer: ```suggestion
You can make updates to a profile that is in use by one or more active clusters. Changing a deployed cluster profile will trigger an update to all clusters using the cluster profile. For more information, refer to [Update a Cluster Profile](../../profiles/cluster-profiles/modify-cluster-profiles/update-cluster-profile.md). 
```
Just some minor clarification.

:::caution
We do not recommend updating a currently deployed cluster profile version to push out changes. Instead, we recommend creating a new profile version, and then upgrade active clusters to the new version. 
:::

</TabItem>

<TabItem value="override-profile-config" label="Override cluster profile configurations">

You can modify the configuration of a deployed cluster without changing the cluster profile itself to update a cluster. 

:::caution

> We do not recommend updating a currently deployed cluster's profile configurations without updating the profile itself. Instead, we recommend creating a new profile version, and then upgrade active clusters to the new version. 

Reviewer: ```suggestion
We do not recommend updating a currently deployed cluster's profile configurations without updating the profile itself. Instead, we recommend creating a new profile version, and then upgrade active clusters to the new version. Check out the [Version a Cluster Profile](../../profiles/cluster-profiles/modify-cluster-profiles/version-cluster-profile.md) for guidance on creating a new cluster profile version.   
```
